### PR TITLE
AddWorkspace endpoint

### PR DESCRIPTION
**Pull request by lalagvana**

### CODE REVIEW

File: src/MessageService.Models.Dto/AddWorkspaceRequest.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace LT.DigitalOffice.MessageService.Models.Dto
+{
+    public class AddWorkspaceRequest
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string Image { get; set; }
```
Comment (djino97) [inline src/MessageService.Models.Dto/AddWorkspaceRequest.cs:7]: What is Image?

File: src/MessageService.Broker/MessageService.Broker.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.9" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.11" />
```
[the remark this answers is not in the document]
Comment (djino97) [reply src/MessageService.Broker/MessageService.Broker.csproj]: fix

**Review by djino97 — APPROVED** [review]

File: src/MessageService.Models.Db/DbWorkspace.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.MessageService.Models.Db
+{
+    public class DbWorkspace
+    {
+        [Key]
+        public Guid Id { get; set; }
```
Comment (KanashinDmitry) [inline src/MessageService.Models.Db/DbWorkspace.cs:9]: Should we use FluentAPI here?

File: test/MessageService.Business.UnitTests/AddWorkspaceCommandTests.cs [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        [Test]
+        public void ShouldAddEmailTemplateCorrectly()
+        {
+            validatorMock
+                .Setup(x => x.Validate(It.IsAny<IValidationContext>()))
+                .Returns(validationResultIsValidMock.Object);
+
+            mapperMock
+                .Setup(mapper => mapper.Map(It.IsAny<AddWorkspaceRequest>()))
+                .Returns(dbWorkspace);
```
Comment (KanashinDmitry) [inline test/MessageService.Business.UnitTests/AddWorkspaceCommandTests.cs:78]: Should we use .Verifiable(), when using just .Verify() without arguments further?

**Review by KanashinDmitry — CHANGES_REQUESTED** [review]

File: src/MessageService.Models.Db/DbWorkspace.cs (hunk 3 above)
Comment (djino97) [reply src/MessageService.Models.Db/DbWorkspace.cs:9]: Why? There are not relationship between tables

File: test/MessageService.Business.UnitTests/AddWorkspaceCommandTests.cs (hunk 4 above)
Comment (djino97) [reply test/MessageService.Business.UnitTests/AddWorkspaceCommandTests.cs:78]: fixed

**Review by MariyaSin — APPROVED** [review]

**Review by KanashinDmitry — APPROVED** [review]

**Pull request merged**